### ISSUE TITLE
`.PortalView` element needs to have height ≠ 100%

### ISSUE THREAD

**Issue by iannesbitt** [NCEAS/metacatui#2265]
**Describe the bug**
A [recent change](https://github.com/NCEAS/metacatui/blame/cd2e5f6834e128f0b5a4786e6893e0b1b0f2d866/src/css/metacatui-common.css#L4879) set the `.PortalView` `height: 100%`, I imagine to keep Cesium pages limited to 100% height. This causes problems on portal pages that are longer than 100%, however, as the footer ends up floating over the body text (see screenshot). To solve this, the `height: 100%` definition needs to be moved to a CSS element that is only applied to pages with a Cesium map.

**To Reproduce**
1. Enable `develop` branch in metacatui
2. Enable `arctic` or `drp` theme
3. Go to a portal with a long markdown page (`localhost:3000/portals/permafrost/About` or `localhost:3000/portals/vt`) depending on the theme
4. Scroll down past footer
5. See footer floating above text

**Expected behavior**
The footer should fall below text.

**Screenshots**
![image](https://github.com/NCEAS/metacatui/assets/18689918/7567738a-88b1-4154-9c9b-23f869aa2040)

**Desktop (please complete the following information):**
all

**Smartphone (please complete the following information):**
all

**Additional context**
n/a

**Comment by iannesbitt** [comment]
It looks like this issue occurs because there is no class that differentiates Cesium portal pages from Markdown pages. I think a fix for this that keeps both the Cesium layer list scroll bar functionality and also pushes the footer down the page would have to involve using JS to add or substitute a class to the `<body>` definition that would indicate the page type. Then that class could be assigned a `height: 100%` or `height: unset` based on needs.

**Comment by iannesbitt** [comment]
Although perhaps there is a way to unset the body height on the Markdown pages directly via script tags...?

**Comment by iannesbitt** [comment]
Putting `body {height:100%}` definitions in `map-view.css` or in the Cesium HTML templates ends up causing this problem too. I think this needs a JS solution.

**Comment by iannesbitt** [comment]
I've tried a few things `PortalVisualizationsView`. I can get `height: 100%` added to the style of `<body>` but I can't figure out how to remove it when the user navigates away from the map page to another portal page. The following works in [this `else if` clause](https://github.com/NCEAS/metacatui/blob/develop/src/js/views/portals/PortalVisualizationsView.js#L134-L137) to add the style definition:
```js
              else if( this.model.get("visualizationType") == "cesium" ){
                $('.PortalView').css('height', '100%')
                this.renderMap();
                this.hasRendered = true;
              }
```

I think the issue with removal is that the [`onClose` function](https://github.com/NCEAS/metacatui/blob/develop/src/js/views/portals/PortalVisualizationsView.js#L177-L179) doesn't work or isn't triggered for whatever reason.

Seems like one solution would be to add logic to `PortalView` that determines the page is not a cesium page, and removes or unsets the height definition.

**Comment by iannesbitt** [comment]
My solution to this is a little different than the above method, instead of using the `.PortalView` element, I set `height: 100%` on `document.style.body` in `PortalVisualizationsView`, and remove the property in `PortalSectionView`. I did this because it echoes the way [the `--footer-height` is set and unset](https://github.com/NCEAS/metacatui/blob/758ff8e3710637d9fdf406a9c2383e56ff5be877/src/js/views/FooterView.js#L30-L42). If it would be cleaner for me to do this another way (such as setting this on `.PortalView`), I'm happy to switch to a different method.
